extract/raw_array_cohort_extract.wdl

```diff
@@ -0,0 +1,182 @@
+version 1.0
+
+workflow RawArrayCohortExtract {
+   input {
+        Int number_of_partitions = 2
+        Int probes_per_partition = 1000000
+        
+        File reference
+        File reference_index
+        File reference_dict
+    
+        String fq_dataset
+        Int max_tables
+        String fq_destination_dataset
+        String query_project
+        String fq_cohort_mapping_table
+        Int ttl
+        
+        String output_file_base_name="export"
+        
+        String full_sample_info_table
+    }
+    
+    call CreateExtractTable {
+        input:
+            fq_dataset                = fq_dataset,
+            max_tables                = max_tables,
+            fq_destination_dataset    = fq_destination_dataset,
+            query_project             = query_project,
+            fq_cohort_mapping_table   = fq_cohort_mapping_table,
+            ttl                       = ttl,
+            number_of_partitions      = number_of_partitions,
+            probes_per_partition      = probes_per_partition
+    }
+  
+    
+    scatter(i in range(number_of_partitions)) {
+        call ExtractTask {
+            input:
+                reference             = reference,
+                reference_index       = reference_index,
+                reference_dict        = reference_dict,
+                probe_info_clause     = "--probe-info-table spec-ops-aou.aou_pmi_synthetic_100k.probe_info",
```

Comment:
good point.  I had it this way because I think we're going to have this read from a bucket... but I probably could even parameterize it... I'll try that